src/modules/details.js

```diff
@@ -1,123 +1,227 @@
+    header: PropTypes.string,
+    style: PropTypes.object,
+    width: PropTypes.string,
+});
+const columnPropsPropTypesShape = PropTypes.shape({
+    horizontalSpacing: PropTypes.number,
+});
 
-        return this._renderColumn(column);
+function ColumnContainer(props) {
+    const {
+        column: {
+            accessor,
+            divide,
+            flexBasis,
+            flexGrow,
+            flexShrink,
+            fontSize,
+            fontWeight,
+            header,
+            style,
+            width,
+        },
+        columnNumber,
+        columnProps,
+        data,
+        moduleType,
+    } = props;
+    const containerClasses = ClassNames(`${moduleType}--details-column${columnNumber ? '-inner' : ''}`, {
+        'divide-left': divide || divide === 'left',
+        'divide-right': divide === 'right',
+    });
+    const accessorClasses = ClassNames(`${moduleType}--details-column-accessor`, {
+        'font-size-large': fontSize === 'large',
+        'font-size-medium': fontSize === 'medium',
+        'font-size-small': !fontSize || fontSize === 'small',
+        'font-weight-bold': !fontWeight || fontWeight === 'bold',
+        'font-weight-normal': fontWeight === 'normal',
+        'font-weight-semibold': fontWeight === 'semibold',
+    });
+    let horizontalSpacing;
+
+    if (columnProps) {
+        horizontalSpacing = columnProps.horizontalSpacing;
     }
 
-    _renderColumn(column, innerContainerKeyNum) {
-        const { columnProps, data, moduleType } = this.props;
-        const containerClasses = ClassNames(`${moduleType}--details-column${!!innerContainerKeyNum ? '-inner' : ''}`, {
-            'divide-left': column.divide || column.divide === 'left',
-            'divide-right': column.divide === 'right',
-        });
-        const accessorClasses = ClassNames(`${moduleType}--details-column-accessor`, {
-            'font-size-large': column.fontSize === 'large',
-            'font-size-medium': column.fontSize === 'medium',
-            'font-size-small': !column.fontSize || column.fontSize === 'small',
-            'font-weight-bold': !column.fontWeight || column.fontWeight === 'bold',
-            'font-weight-normal': column.fontWeight === 'normal',
-            'font-weight-semibold': column.fontWeight === 'semibold',
-        });
-        let accessor, flexBasisInlineStyle, flexGrowInlineStyle, flexShrinkInlineStyle, horizontalSpacingInlineStyle;
+    let newAccessor;
+    let flexBasisInlineStyleValue;
+    let flexGrowInlineStyleValue;
+    let flexShrinkInlineStyleValue;
+    let horizontalSpacingInlineStyleValue;
 
-        if (_.isString(column.accessor)) {
-            accessor = _.get(data, column.accessor);
-        } else if (_.isFunction(column.accessor)) {
-            accessor = column.accessor(data);
-        }
+    if (_.isString(accessor)) {
+        newAccessor = _.get(data, accessor);
+    } else if (_.isFunction(accessor)) {
+        newAccessor = accessor(data);
+    }
```

Comment:
`newAccessor` should be `accessedData` maybe?  i.e. it's the value we've extracted from `data` using the `accessor`.